domain/service/api.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"github.com/control-center/serviced/datastore"
+	"github.com/control-center/serviced/validation"
+)
+
+// Details is the minimal data necessary to show for a service
+type Details struct {
```

Comment:
I think this file should be called servicedetails.go and the _store.go file should be named servicedetails_store.go